public/app/features/dashboard/components/SaveDashboard/forms/SaveDashboardForm.tsx

```diff
@@ -84,8 +84,21 @@ export const SaveDashboardForm = ({
               aria-label={selectors.pages.SaveDashboardModal.saveVariables}
             />
           )}
-
-          <TextArea {...register('message')} placeholder="Add a note to describe your changes." autoFocus rows={5} />
+          <TextArea
+            {...register('message')}
+            aria-label="message"
+            value={options.message}
+            onChange={(e) => {
+              onOptionsChange({
+                ...options,
+                message: e.currentTarget.value,
+              });
+              register('message').onChange(e);
```

Comment:
Instead of calling `register` twice, please define it as const within the render prop closure, L60.

Reply:
> Instead of calling `register` twice, please define it as const within the render prop closure, L60.

Thank you for your help!
hope I understand you in the right way :)

Reply:
What you did is exactly what I had in mind. Thank you!

Reply:
Thank you! Should I do something to merge it?